packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts

```diff
@@ -37,7 +25,7 @@ const { getContractFactory } = hre.ethers;
 async function main() {
   console.log("Running VotingV2 Deployments🔥");
 
-  const networkId = Number(await hre.getChainId());
+  const networkId = await hre.ethers.provider.getNetwork().then((network) => network.chainId);
```

Comment:
Minor nit: prefer chainId terminology, as networkId is technically a different concept (although they _usually_ match).